code/modules/clothing/glasses/glasses.dm

```diff
@@ -87,6 +87,34 @@
 		if(istype(A, /datum/action/item_action/toggle))
 			A.update_button_icon()
 
+/obj/item/clothing/glasses/proc/try_make_offhand_prescription(mob/user)
+	if(!prescription)
+		return FALSE
+
+	var/obj/item/clothing/glasses/offhand = user.get_inactive_hand()
+	if(istype(offhand) && !offhand.prescription)
+		if(tgui_alert(user, "Do you wish to take out the prescription lenses and put them in [offhand]?", "Insert Prescription Lenses", list("Yes", "No")) == "Yes")
+			offhand.prescription = TRUE
```

Comment:
I intended to have them guard against this, but I just didn't place it in the correct spot.
![image](https://github.com/cmss13-devs/cmss13/assets/76988376/b30a0b26-6616-47d4-ba0b-3c93a104aba6)

```suggestion
			if(QDELETED(src) || offhand != user.get_inactive_hand())
				return FALSE
			offhand.prescription = TRUE
```